dkg/pedersen/handler.go

```diff
@@ -670,12 +675,12 @@ func (h *Handler) handleDeal(ctx context.Context, msg types.Deal,
 	return nil
 }
 
-func (h *Handler) finalizeReshare(ctx context.Context, isCommonNode bool, out mino.Sender, from mino.Address) error {
+func (h *Handler) finalizeReshare(ctx context.Context, nodeType int, out mino.Sender, from mino.Address) error {
```

Comment:
would be even better to declare a type:

```
type nodeType byte
const (
    unknownNode nodeType = iota
    oldNode
    commonNode
...
)
...
func (h *Handler) finalizeReshare(ctx context.Context, t nodeType, out mino.Sender, from mino.Address) error {
```

Reply:
Didn't know, but agree. What a cool language. thanks